### ISSUE TITLE
NTR: Glycylproline high in blood (Lab: chemistry: amino compounds)

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#8551]
New term request Glycylproline high in blood (elthat_111113012436):

**Comment by pnrobinson** [comment]
@MickeySegal
am I correct this should be a child of [Abnormal circulating collagen degradation product concentration HP:0033386](https://hpo.jax.org/app/browse/term/HP:0033386)
with
label: Elevated circulating glycylproline concentration
definition: Concentration in the bloodstream of glycylproline above the upper limit of normal. Glycylproline is a dipeptide composed of glycine and proline, and is an end product of collagen metabolism that is further cleaved by prolidase (EC 3.4.13.9). 

This is a feature of 

Prolidase deficiency (OMIM:170100)

**Comment by MickeySegal** [comment]
We do have this for prolidase deficiency.  BTW, https://omim.org/entry/170100 uses glycyl-L-proline and glycylproline interchangeably so it makes sense to include both.

**Comment by pnrobinson** [comment]
@MickeySegal 
Adding new term: 
- ID: HP:6000271
- label: Elevated circulating glycylproline concentration
- parents: [Abnormal circulating collagen degradation product concentration (HP:0033386)](https://hpo.jax.org/app/browse/term/HP:0033386)
- pmid: 
- def: Concentration in the bloodstream of glycylproline above the upper limit of normal. Glycylproline is a dipeptide composed of glycine and proline, and is an end product of collagen metabolism that is further cleaved by prolidase (EC 3.4.13.9).
- comment: 
- orcid: orcid:0000-0002-0736-9199
- github ID: https://github.com/obophenotype/human-phenotype-ontology/issues/8551
- synonyms:
